ebi_eva_common_pyutils/spring_properties.py

```diff
@@ -121,11 +122,13 @@ def get_accessioning_properties(self, instance='', target_assembly='',  fasta=''
             },
         )
 
-    def get_remapping_extraction_properties(self, *, taxonomy='', source_assembly='', fasta='', assembly_report='',
-                                            projects='', output_folder='.'):
+    def get_remapping_extraction_properties(self, *, taxonomy=None, source_assembly=None, fasta=None,
+                                            assembly_report=None,
+                                            projects=None, output_folder=None):
```

Comment:
```suggestion
                                            projects='', output_folder=None):
```
New target assembly remapping doesn't use projects, so by the same logic as for the clustering properties I think this can default to `''`.